todo-app/src/TodoStatus.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+export default class TodoStatus extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      done: this.props.done || false,
+    }
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({
+      done: !this.state.done
```

Comment:
Updating the state with `setState` is the way to go, nice 👍 You could have used the `checked` property of the event target, `event.target.checked`. So you don't depend on the previous value of the state. You might run in problems with this, as the state [updates asynchronously](https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous).